Tests/FluentAssertions.Specs/FluentAssertions.Specs.csproj

```diff
@@ -9,7 +9,6 @@
     <IsPackable>false</IsPackable>
     <CodeAnalysisRuleSet>..\..\TestRules.ruleset</CodeAnalysisRuleSet>
     <!-- https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA0001.md -->
-    <DocumentationFile Condition="'$(TargetFramework)' == 'net5.0'">$(OutputPath)$(AssemblyName).xml</DocumentationFile>
```

Comment:
Why is this removed?

Reply:
Because we don't need to analyze XML comments in test projects. I also disabled rule 5001 for that. I was thinking whether to put that in a separate PR, but couldn't be bothered. Should have known you would ask about that :-)

Reply:
Roger, makes perfect sense 👍 

Then please remove the comment linking to `SA0001.md` from this and the new specs project.